### ISSUE TITLE
There shouldn't be empty buttons on the web interface

### ISSUE THREAD

**Issue by mgifford** [RocketChat/Rocket.Chat#10379]
### Description: 

Empty buttons present accessibility problems for people with disabilities. These should be explicitly hidden or not present. 

This presents a problem for WCAG 2.0 AA requirements.

### Steps to Reproduce:

1. Load up a rocket chat in a web browser
2. Run the wave toolbar http://wave.webaim.org/extension/
3. Look for all of the red boxes

### Expected behavior:

There should be no empty buttons or links. 

### Actual behavior:

There are empty buttons

<img width="368" alt="screenshot from webaim" src="https://user-images.githubusercontent.com/116832/38456655-f9b7d780-3a54-11e8-8c98-b85792af5c02.png">

**Comment by Cambridgeport90** [comment]
Thanks @mgifford. I just want to also point out the fact that this is present no matter which interface you use; hardly any of the buttons are labeled at all; I am blind myself, using text to speech. I get lots of "button" and "clickable" when navigating throughout the interface because no screen reader can see behind the buttons and other controls with no labels. I'll pop up a menu and still not figure out how I did it. This is a huge problem.

**Comment by mgifford** [comment]
Thanks @Cambridgeport90 - I appreciate you verifying this issue. The first step to fix these problems is to report them.  Point to the W3C standards where the guidelines are laid out.  It doesn't always work, but if enough people do it. 

Happy to help try work with you to identify some of these.

**Comment by HammyHavoc** [comment]
If I have some free time in the next few days then I'll attempt to tackle this for you, @Cambridgeport90. I work with several legally blind users, there's a lot of work to be done on Rocket.Chat in regards to being accessible. One user has some sight, though the bright white UX makes Rocket.Chat impossible to use.

Currently reading through the Web Accessibility Initiative on W3C.

**Comment by graywolf336** [comment]
If anyone submits a pull requests to fix these issues, please tag me. If they're in before our cut off date for the next release, which is the 20th of each month, then I'll try my best to ensure they get merged. I say let's start small and incrementally fix them instead of trying to fix everything in one run.

**Comment by Cambridgeport90** [comment]
Agreed. Work up to completely accessible while in the short run making it useable.

**Comment by mgifford** [comment]
It's a journey, not a destination. There are always ways of making things more usable & accessible.  This one is just low-hanging-fruit. Should be fairly straight forward.

**Comment by Cambridgeport90** [comment]
Please. Keep me updated. I love your enthusiasm. If I knew electron, I’d lend a hand directly.

**Comment by mgifford** [comment]
I still see it in the instance I use. But I don't maintain it and am not sure what version of the software it is using. I assume it is updated.

**Comment by geekgonecrazy** [comment]
👍 sounds like still is an issue then

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and maintained releases.

[Rocket.Chat](https://rocket.chat/) provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
